packages/node_modules/@webex/plugin-meetings/src/statsAnalyzer/index.js

```diff
@@ -295,6 +480,11 @@ export default class StatsAnalyzer extends EventsScope {
    * @memberof StatsAnalyzer
    */
   processTrackResult(result, mediaType) {
+    if (!result) {
```

Comment:
Let's combine all these empty checks into one:

```suggestion
    if (!result || result.type !== 'track') {
```